start_pokealarm.py

```diff
@@ -684,16 +684,16 @@ def parse_settings(root_path):
         managers[m_name].start()
 
     # Set up signal handlers for graceful exit
-    signal(signal.SIGINT, exit_gracefully)
-    signal(signal.SIGTERM, exit_gracefully)
+    signal_handler(signal.SIGINT, exit_gracefully, signal.SIGINT, None)
+    signal_handler(signal.SIGTERM, exit_gracefully, signal.SIGTERM, None)
 
 
 # Because lists are dumb and don't have a failsafe get
 def get_from_list(arg, i, default):
     return arg[i] if len(arg) > 1 else default
 
 
-def exit_gracefully():
+def exit_gracefully(signum, frame):
     log.info("PokeAlarm is closing down!")
```

Comment:
maybe add a debug log entry here as to which SigNum we are responding to since we have it available now.